buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -10,16 +10,16 @@ import java.util.Locale
 // https://kotlinlang.org/docs/reference/whatsnew15.html
 // https://kotlinlang.org/docs/releases.html#release-details
 // TODO: Update corresponding buildSrc/build.gradle.kts value when updating this version!
-private const val KOTLIN_VERSION = "1.7.0"
-private const val KOTLIN_COROUTINES_VERSION = "1.6.0"
+private const val KOTLIN_VERSION = "1.7.20"
+private const val KOTLIN_COROUTINES_VERSION = "1.6.4"
 
 /**
  * Provides the source of truth for version/configuration information to any gradle build file (project and app module build.gradle.kts)
  */
 object Config {
     // https://github.com/JLLeitschuh/ktlint-gradle/blob/master/CHANGELOG.md
     // https://github.com/JLLeitschuh/ktlint-gradle/releases
-    const val KTLINT_GRADLE_VERSION = "10.2.1"
+    const val KTLINT_GRADLE_VERSION = "11.0.0"
```

Comment:
@LuisRocketRangel Please run the Android Studio `ktlintFormat` Run Configuration to see if any additional violations are found with the latest version of ktlint